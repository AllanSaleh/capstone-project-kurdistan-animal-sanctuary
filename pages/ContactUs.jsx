import React from "react";
import { serverSideTranslations } from "next-i18next/serverSideTranslations";
import Navbar from "../components/Navbar";
import Contacts from "../components/ContactUsBody";
import Footer from "../components/Footer";

export default function Home() {
  // let name = 'Allan';
  return (
    <>
      <Navbar />

      <Contacts />

      <Footer />
    </>
  );
}

export async function getStaticProps({ locale }) {
  return {
    props: {
      ...(await serverSideTranslations(locale, [
        "common",
        "navbar",
        "contactus",
        "footer",
      ])),
      // Will be passed to the page component as props
    },
  };
}
