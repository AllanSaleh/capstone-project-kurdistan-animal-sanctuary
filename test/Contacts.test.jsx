import React from "react";
import renderer from "react-test-renderer";
import { test, expect } from "@jest/globals";
import ContactUs from "../components/ContactUsBody";

test("proparly renders the ContactUs component", () => {
  const tree = renderer.create(<ContactUs />).toJSON();
  expect(tree).toMatchSnapshot();
});
