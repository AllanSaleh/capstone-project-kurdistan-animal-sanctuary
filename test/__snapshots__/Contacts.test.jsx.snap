// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`proparly renders the ContactUs component 1`] = `
<div
  className="flex flex-col items-center w-full pb-10"
>
  <h1
    className="text-center mt-10 text-7xl text-highlight"
  >
    contactus.ContactUs
  </h1>
  <p
    className="text-center mt-12 text-primary"
  >
    contactus.houseWithoutFlower
  </p>
  <p
    className="text-center text-primary"
  >
    contactus.sendUsurNote
  </p>
  <form
    action="#"
    className="flex flex-col items-center w-4/6"
  >
    <div
      className="flex flex-col justify-between w-full mt-10 lg:flex-row"
    >
      <div
        className="flex flex-col"
      >
        <label
          className="pl-1.5 text-primary text-xl mb-1"
          htmlFor="FullName"
        >
          contactus.Fullname
        </label>
        <input
          className="border border-primary placeholder-primary rounded-2xl h-11 w-full lg:w-56 bg-secondary placeholder-opacity-100 pl-1.5 box-border outline-none"
          id="FullName"
          placeholder="contactus.Fullname"
          type="text"
        />
      </div>
      <div
        className="flex flex-col"
      >
        <label
          className="pl-1.5 text-primary text-xl mb-1"
          htmlFor="email"
        >
          contactus.Email
        </label>
        <input
          className="border border-primary placeholder-primary rounded-2xl h-11 w-full lg:w-56 bg-secondary placeholder-opacity-100 pl-1.5 box-border outline-none"
          id="email"
          placeholder="contactus.Email"
          type="email"
        />
      </div>
      <div
        className="flex flex-col"
      >
        <label
          className="pl-1.5 text-primary text-xl mb-1"
          htmlFor="phoneNumber"
        >
          contactus.Phone
        </label>
        <input
          className="border border-primary placeholder-primary rounded-2xl h-11 w-full lg:w-56 bg-secondary placeholder-opacity-100 pl-1.5 box-border outline-none"
          id="phoneNumber"
          placeholder="contactus.Phone"
          type="tel"
        />
      </div>
    </div>
    <textarea
      className="border border-primary placeholder-primary mt-5 rounded-2xl resize-none bg-secondary placeholder-opacity-100 pl-1.5 pt-1 box-border outline-none w-full"
      name="message"
      placeholder="contactus.Help"
      rows="10"
    />
    <button
      className="mt-5 rounded-2xl w-28 h-10 text-primary bg-highlight active:bg-primary active:text-highlight font-medium"
      type="submit"
    >
      contactus.Send
    </button>
  </form>
</div>
`;
